pkg/code-generator/parser/parser_resource.go

```diff
@@ -59,40 +52,34 @@ func getResource(resources []*model.Resource, project model.ProjectConfig, cfg m
 	case 0:
 		return nil, errors.Errorf("getting resource: message %v not found", cfg)
 	}
-	// default to using the version matching the project itself
-	// only works for this project's resources
-	for _, res := range possibleResources {
-		if res.GoPackage == project.GoPackage {
-			return res, nil
-		}
-	}
-	return nil, errors.Errorf("found %v resources found which match %v, try specifying a version", len(possibleResources), cfg)
+
+	return possibleResources[0], nil
```

Comment:
The above switch statement is no longer necessary as really you are only checking for empty, so you can just do that in an if